bubble-wrap-style.yaml

```diff
@@ -629,21 +629,19 @@ styles:
                 global: |
                     #ifdef TANGRAM_FRAGMENT_SHADER
                         float TileDots(float scale, float size) {
+
+                            // controls
+                            float DOT_SIZE = 1.8; // bigger value = smaller dots
+                            float SPEED = 10.; // bigger value = faster transition
+
                             vec2 tc = TileCoords() * scale * pow(2., floor(u_map_position.z) - abs(u_tile_origin.z));
```

Comment:
While we're cleaning up this shader, can we swap the `pow(2., ...` for an `exp2( ...`?

Reply:
good idea!